src/components/Header/index.jsx

```diff
@@ -28,20 +28,19 @@ function Header({ schema, sourceMode, toggleMode }) {
 
   return (
     <div className={classes.container}>
-      <Typography
-        component="div"
-        variant="h6"
-        className={classes.title}>
+      <Typography component="div" variant="h6" className={classes.title}>
         {schema.title}
         {` `}
-        <Button className={classes.button} color="inherit" size="small" onClick={() => toggleMode()}>
+        <Button
+          className={classes.button}
+          color="inherit"
+          size="small"
+          onClick={() => toggleMode()}>

```

Comment:
This could just be `onClick={toggleMode}`.

Reply:
Applied changes!